From 17f834499b73712480e6eb97c0c40ab26f947499 Mon Sep 17 00:00:00 2001
From: Omeshwari wazekar <quic_owazekar@quicinc.com>
Date: Wed, 29 Mar 2023 09:33:17 +0530
Subject: [PATCH 279/281] dt-bindings: watchdog: qcom-wdt: add
 qcom,apss-wdt-ipq5332 compatible

Add a compatible for the IPQ5332 platform's APSS watchdog.

Change-Id: I7ac461436771c15d8a4b4c369c07c21d334b0350
Signed-off-by: Omeshwari wazekar <quic_owazekar@quicinc.com>
---
 Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml b/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml
index 295500153f67..259f444351c1 100644
--- a/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml
+++ b/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml
@@ -17,6 +17,7 @@ properties:
     oneOf:
       - items:
           - enum:
+              - qcom,apss-wdt-ipq5332
               - qcom,apss-wdt-ipq9574
               - qcom,apss-wdt-qcs404
               - qcom,apss-wdt-sc7180
-- 
2.17.1

